packages/neon-core/src/wallet/verify.ts

```diff
@@ -138,3 +138,9 @@ export function isAddress(address: string): boolean {
     return false;
   }
 }
+
+export function isAssetHash(assetHash: string): boolean {
+  return (
+    (isHex(assetHash) && assetHash.length === 40) || assetHash.length === 64

```

Comment:
I think you are mixing up assets and tokens here. Assets are UTXO styled with the 64 char hash. The 40 char hash belongs to tokens.

Reply:
ah ok, so just a "isHex" and "assetHash.lenght === 64" validation are necessary?

Reply:
yups i believe those are the only conditions.

Reply:
ok but i checked the get_balance answer with other tokens like RPX or PHX and the asset_hash of those tokens is 40 characters long, wouldn't that cause a problem?

Reply:
I think there is a misconception here. RPX and PHX are not assets. They are NEP5 tokens and do not use UTXO. Only GAS and NEO are assets so far in neo2.

Reply:
ok i really didn't notice it, I will fix the function, tks